regimes/pl/extensions.go

```diff
@@ -7,8 +7,9 @@ import (
 
 // Regime extension codes for local electronic formats.
 const (
-	ExtKeyKSeFVATZero    = "pl-ksef-vat-zero"
-	ExtKeyKSeFVATSpecial = "pl-ksef-vat-special"
+	ExtKeyKSeFVATZero       = "pl-ksef-vat-zero"
+	ExtKeyKSeFVATSpecial    = "pl-ksef-vat-special"
+	ExtKeyKSEFEffectiveDate = "pl-ksef-effective-date"
```

Comment:
Based on [Greg's this description](https://github.com/invopop/gobl/pull/254#discussion_r1528608583), I think this is a good name for the extension. The only (minor) drawback is that the name could suggest that the content will be an actual date. We could avoid that by making the name longer but I think it's preferable to keep it succinct. I wouldn't change it.

I spotted a typo in the constant name, though:
```suggestion
	ExtKeyKSeFEffectiveDate = "pl-ksef-effective-date"
```